types/node/tls.d.ts

```diff
@@ -114,12 +151,50 @@ declare module "tls" {
          */
         alpnProtocol?: string;
 
+        /**
+         * Returns an object representing the local certificate. The returned
+         * object has some properties corresponding to the fields of the
+         * certificate.
+         *
+         * See tls.TLSSocket.getPeerCertificate() for an example of the
+         * certificate structure.
+         *
+         * If there is no local certificate, an empty object will be returned.
+         * If the socket has been destroyed, null will be returned.
+         */
+        getCertificate(): PeerCertificate | {} | null;
```

Comment:
Same here, should probably be `object`.

Reply:
Ok, it looks like it should be `object` instead `{}` really. Then I'll get

```
Property 'name' does not exist on type 'object | EphemeralKeyInfo'.
  Property 'name' does not exist on type 'object'.ts(2339)
```

without proper type guard.